### DISCUSSION TITLE
Settings tablet play area by using pen

### DISCUSSION THREAD

**Discussion by M4xi1m3** [ppy/osu#13192] [Ideas]
**Describe the new feature:** 
We should be able to set the play zone of the in-game tablet driver by setting for example the top left and bottom right corner by using the pen, instead of using sliders.
**Proposal designs of the feature:** 
A button "Select area..." which would open a pop-up menu asking you to click on the top left corner of the desired play area and then asking you to click on the bottom right corner, for example.

**Comment by peppy** [comment]
A wizard for setting things up is planned in a future pass.
